brayns/Brayns.cpp

```diff
@@ -191,7 +191,16 @@ struct Brayns::Impl : public PluginAPI
         _extensionPluginFactory.preRender();
 
         auto& scene = _engine->getScene();
+        auto& camera = _engine->getCamera();
+
+        if (_parametersManager.isAnyModified() || camera.isModified() ||
+            scene.isModified() || scene.getTransferFunction().isModified())
```

Comment:
Can you test if a markModified() in commitTransferFunctionData() would also work, so that the scene.isModified() is enough here?

Reply:
OK